Update branch/19.08 to new releases

cxrvh opened the pull request:
This updates the manifest to a yaml (as was done in the other openjdk extensions) & updates all releases to the newest version.

Since i386 was dropped by the Freedesktop SDK 19.08 & to simplify the manifest, the bootstrap-java was replaced from the Debian packages to the AdoptOpenJDK releases which provide all necessary architectures.

**The update was only built & tested for x86_64!**

Commit:
Convert json to yaml

Commit:
Update bootstrap-java to newest AdoptOpenJDK release

Update the bootstrap-java & since support for i386 was dropped with
Freedesktop SDK 19.08, replace the Debian packages with the AdoptOpenJDK
releases to simplify the built. AdoptOpenJDK supports all needed
architectures.

Commit:
Update to jdk8u265-b01

Commit:
Update maven to 3.6.3

A6GibKm commented:
- I think `--with-company-name` was replaced with `--with-vendor-name` but I am not sure on this
- If you add sources for `arm` consider removing the `skip-arches` from `flathub.json` file

Commit:
Split company-/vendor-name configuration by arch

The aarch64 repository only supports the vendor-prefixed options of
newer openjdk releases.

Commit:
Remove skipped arm sources

Commit:
Format yaml

cxrvh commented:
@MatMaul @TingPing 

Would be great if someone could review & merge the changes here and for branch/20.08 (#9). Or should I just create an issue in the general flathub project?

MatMaul commented:
Thanks for that, LGTM. Let's merge and publish the 20.08 branch.
